ITEM.tsv

```diff
@@ -6069,9 +6069,9 @@ ITEM_20150918_006068	Krag Pike
 ITEM_20150918_006069	Pilgrim Pike
 ITEM_20150918_006070	Istora Pike
 ITEM_20150918_006071	Arde Dagger
-ITEM_20150918_006072	
+ITEM_20150918_006072	The fire sprouting from this blade doesn't affect the user, but may instead catch onto enemies.
 ITEM_20150918_006073	Karacha Dagger
-ITEM_20150918_006074	
+ITEM_20150918_006074	 - Every attack from this dagger has a 2% chance of inflicting poison on an enemy that deals 6165 damage to it over the course of 10 seconds.
```

Comment:
I have: 

[Poison Effect] 2% chance to inflict 6165 damage every second over 10 seconds to an enemy